docs/S3ToRedshift-action.md

```diff
@@ -0,0 +1,68 @@
+# S3ToRedshift Action
+
+
+Description
+-----------
+S3ToRedshift Action that will load the data from AWS S3 bucket into the AWS Redshift table.
+
+Properties
+----------
+
+**accessKey:** Access key for AWS S3 to connect to. (Macro-enabled)
+
+**secretAccessKey:** Secret access key for AWS S3 to connect to. (Macro-enabled)
+
+**iamRole:** IAM Role for AWS S3 to connect to. (Macro-enabled)
```

Comment:
Could you add "this can only be used if your cluster is being hosted on AWS servers?